docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.install

```diff
@@ -246,3 +246,92 @@ function hs_paragraph_types_update_10006() {
     }
   }
 }
+
+/**
+ * Set new config for color wells.
+ */
+function hs_paragraph_types_update_10007(&$sandbox) {
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  if (empty($sandbox['ids'])) {
+
+    $config_storage = \Drupal::service('config.storage.sync');
+    $config_factory = \Drupal::configFactory();
+
+    /** @var \Drupal\Core\Entity\EntityStorageInterface $field_config_storage */
+    $field_config_storage = \Drupal::service('entity_type.manager')->getStorage('field_config');
+
+    /** @var \Drupal\Core\Entity\EntityStorageInterface $fieldConfigStorage */
+    $field_storage = \Drupal::service('entity_type.manager')->getStorage('field_storage_config');
+
+    // If storage does not yet exist, create it.
+    $storage_to_create = [
+      'paragraph.field_bg_color' => 'field.storage.paragraph.field_bg_color',
+      'paragraph.field_bg_color_width' => 'field.storage.paragraph.field_bg_color_width',
+    ];
+
+    foreach ($storage_to_create as $key => $fields) {
+      if (empty($field_storage->load($key))) {
+        $field_storage->createFromStorageRecord($config_storage->read($fields))->save();
+      }
+    }
+
+    $field_instances = [
+      'paragraph.hs_collection.field_bg_color' => 'field.field.paragraph.hs_collection.field_bg_color',
+      'paragraph.hs_collection.field_bg_color_width' => 'field.field.paragraph.hs_collection.field_bg_color_width',
+    ];
+
+    // Create the field instance.
+    foreach ($field_instances as $key => $instance) {
+      $config = $field_config_storage->load($key);
+      if (empty($config)) {
+        $field_config_storage->create($config_storage->read($instance))->save();
+      }
+    }
+
+    $displays = [
+      'core.entity_view_display.paragraph.hs_collection.default',
+      'core.entity_form_display.paragraph.hs_collection.default'
+    ];
+
+    foreach ($displays as $display) {
+      // Update displays.
+      $config = $config_factory->getEditable($display);
+      $new_config_data = $config_storage->read($display);
+      $config->setData($new_config_data);
+      $config->save();
+    }
+
+    // Get list of collection paragraphs.
+    $sandbox['ids'] = $paragraph_storage->getQuery()
+      ->accessCheck(FALSE)
+      ->condition('type', 'hs_collection')
+      ->execute();
+    $sandbox['total'] = count($sandbox['ids']);
+  }
+
+  $paragraph_ids = array_splice($sandbox['ids'], 0, 10);
+
+  // Set new field default for existing hs_collection paragraphs.
+  foreach ($paragraph_storage->loadMultiple($paragraph_ids) as $paragraph) {
+    /** @var \Drupal\paragraphs\Entity\Paragraph $paragraph */
+    if (
+      $paragraph->hasField('field_paragraph_style') &&
+      $paragraph->hasField('field_bg_color') &&
+      $paragraph->hasField('field_bg_color_width')) {
+      // Only set new field if old field had data.
+      if ($paragraph->get('field_paragraph_style')->value) {
```

Comment:
To reduce your complexity score you could simplify to 
```php
$paragraph->field_paragraph_style->value
```
Because of your checks above, you know that this will be safe.

Same with all other lines deeper down.